src/deadline/client/cli/_common.py

```diff
@@ -182,3 +189,42 @@ def _cli_object_repr(obj: Any):
     # strings to end with "\n".
     obj = _fix_multiline_strings(obj)
     return deadline_yaml_dump(obj)
+
+
+class _ProgressBarCallbackManager:
```

Comment:
This seems to be refactored out of the bundle group from Sean's code base.

Reply:
`_` signifies internal classes, prepend all classes we do not want customers to use with `_`